components/event-sources/Makefile

```diff
@@ -53,13 +56,16 @@ vendor: Gopkg.lock
 	@echo '+ Pulling vendored dependencies'
 	@dep ensure -v --vendor-only
 
+$(patched_trace): hack/set-span-id.patch vendor
```

Comment:
Your patch is _modifying_ an existing file if I'm not mistaken, which means your target must be PHONY, otherwise `make` will report "xyz is up-to-date."